spec/features/users/show_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'User Dashboard', type: :feature do
+   it 'shows users names dashboard at top of page' do
+      user_1 = User.create!(name: 'First User', email: 'firstuser@mail.com')
+      user_2 = User.create!(name: 'Second Visitor', email: 'secondvisitor@email.com')
+      
+      visit "/users/#{user_1.id}"
```

Comment:
I have been using the rails routes path syntax `user_path(user_1)`. Which syntax for routes to you prefer so that we are consistent in our project?

Reply:
Thanks for fixing this!